recipe/meta.yaml

```diff
@@ -132,10 +134,12 @@ requirements:
     - freetype                           # [linux]
     - harfbuzz
     - glib
-    - gst-plugins-base  {{ gstreamer }}  # [unix]
+    # QtMultimedia will be split off to a separate feedstock after 6.7.1
+    - gst-plugins-base  {{ gstreamer }}  # [unix and version == "6.7.1"]
     # gstreamer plugin not supported on win32: https://bugreports.qt.io/browse/QTBUG-107073
-    - gstreamer                          # [unix]
-    - pulseaudio                         # [linux]
+    - gstreamer                          # [unix and version == "6.7.1"]
+    - ffmpeg                             # [version == "6.7.1"]
```

Comment:
could we actually remove this line and make it only part of the qt-multimedia package.

Reply:
So basically don't use ffmpeg as the backend for 6.7.1, but installing the qt-multimedia package would add ffmpeg support?

Reply:
That would be ideal